### DISCUSSION TITLE
Are the custom PCBs from version 4.0 compatible with those from version 4.1?

### DISCUSSION THREAD

**Discussion by ermolenkodev** [adamb314/ServoProject#25] [Q&A] [answered]
Hi, thank you for open-sourcing this amazing project and for the efforts you have put into providing manuals! I find it truly inspiring.
I have placed an order for PCBs according to the design for version 4.0, and I am curious to know if there have been any incompatible changes. Would it be possible for me to use the version 4.0 PCBs together with the updated manuals for version 4.1?

**Comment by adamb314** [comment, marked as answer]
Thank you!

There haven't been any changes to the combined PCB design between 4.0 -> 4.1, so you can use the updated build instructions :)

Looking forward to any feedback you might have on the build experience!
